### PR TITLE
handler for corrupted text

### PR DESCRIPTION
**Pull request by vzhenia**

### CODE REVIEW

File: src/build/firefox.manifest.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		"*://*.facebook.com/*",
 		"*://whotargets.me/*",
 		"*://*.whotargets.me/*",
-		"*://localhost:*/*"
```
Comment (skinofstars) [inline src/build/firefox.manifest.json:17]: Sorry, I missed this, did it get merged like this? Oops, my bad.

It should be `"*://localhost/*"`. I wouldn't expect to see a `*` for a port range.

File: src/daemon/collector/content/collect.js [hunk 2]
```diff
@@ -241,6 +241,10 @@ function filterFrontAds(lst) {
       && isScrolledIntoView(lst[i], layoutStyle)) {
         newLst.push(lst[i]);
     }
+    if (sponsoredText.includes(lst[i].getAttribute('aria-label'))
```
Comment (skinofstars) [inline src/daemon/collector/content/collect.js:244]: Is this just checking for an `aria-label`? Is that distinguishing enough?

File: src/build/firefox.manifest.json (hunk 1 above)
Comment (vzhenia) [reply src/build/firefox.manifest.json:17]: changed

File: src/daemon/collector/content/collect.js (hunk 2 above)
Comment (vzhenia) [reply src/daemon/collector/content/collect.js]: for now aria-label of the ad contains the text we can check

**Pull request merged**